### DISCUSSION TITLE
Change width of the entire map

### DISCUSSION THREAD

**Discussion by mhechthz** [freeplane/docs#66] [Q&A] [answered]
Sorry, to open this again, but the other question was closed, but not solved. 
![image](https://user-images.githubusercontent.com/37075144/182089653-5b021dea-1c44-4a2a-a201-9b59d5d6bdd7.png)

This view is much too narrow (horizontally) in my opinion. What I liked to see is something like this

![image](https://user-images.githubusercontent.com/37075144/182089986-506c585d-acce-4b93-8d00-acf246e8916e.png)

So the question would be: is there a chance to modify the horizontal expansion of the map. The proposed solution [https://github.com/freeplane/docs/issues/64#issuecomment-1200401057](url) does not work because I don't want to see the outline but the 'normal' map or to have the map more squared than vertical.

**Comment by dpolivaev** [comment, marked as answer]
Please ask your questions in the right place.

https://github.com/freeplane/freeplane/discussions

**Reply by mhechthz** [reply]
Sorry, I thought I am in the discussion section. But obviously not the right one.
